MAUI/StepProgressBar/description.md

```diff
@@ -0,0 +1,413 @@
+
+ public class ViewModel
+ {
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    private ObservableCollection<StepProgressBarItem> stepProgressItem;
+
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    public ObservableCollection<StepProgressBarItem> StepProgressItem
+    {
+        get
+        {
+            return stepProgressItem;
+        }
+        set
+        {
+            stepProgressItem = value;
+        }
+    }
+
+    public ViewModel()
+    {
+        stepProgressItem = new ObservableCollection<StepProgressBarItem>();
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Enter Email", ImageSource = "email.png" });
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Create Password", ImageSource = "password.png" });
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Verify Email", ImageSource = "verified.png" });
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Complete Profile", ImageSource = "profile.png" });
+    }
+ }
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to step progressbar
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="8" %}
+
+<stepProgressBar:SfStepProgressBar
+                    x:Name="stepProgress"
+                    VerticalOptions="Center"
+                    HorizontalOptions="Center"                                        
+                    Orientation="Horizontal"                                                                                       
+                    LabelSpacing="12"
+                    LabelPosition="Bottom"
+                    ActiveStepIndex="2"
+                    ActiveStepProgressValue="50"    
+                    ItemsSource="{Binding StepProgressItem}">
+</stepProgressBar:SfStepProgressBar>                                                                                              
+
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="9" %}
+
+ViewModel viewModel = new ViewModel();
+SfStepProgressBar stepProgressBar = new SfStepProgressBar()
+{
+    VerticalOptions = LayoutOptions.Center,
+    HorizontalOptions = LayoutOptions.Center,
+    Orientation = StepProgressBarOrientation.Horizontal,
+    LabelSpacing = 12,
+    LabelPosition = LabelPosition.Bottom,
+    ActiveStepIndex = 2,
+    ActiveStepProgressValue = 50,
+    ItemsSource = viewModel.StepProgressItem,
+};
+
+this.Content = stepProgressBar;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customization using Font icon
+
+You can add the images with font icons to the [ImageSource]() property in `SfStepProgressBar`. The following code example explains how to set an image in step view.
+
+#### Initialize view model
+
+* Create a simple Observable Collection of item with font icons for the step progressbar,
```

Comment:
* Create a simple Observable Collection of item with font icons for the step progressbar.

Reply:
Content modified